docs/nuget-org/policies/deleting-packages.md

```diff
@@ -34,3 +34,27 @@ Packages that meet any of the following criteria are not allowed on the public N
 If you find a package that is in violation of any of these items, click the **Report Abuse** link on the package details page and submit a report.
 
 Note that the NuGet team and the .NET Foundation reserves the right to change these criteria at any time.
+
+#### Unlisting a package
+In order to unlist a specific package version please follow these steps:
```

Comment:
We try to avoid using "please" in docs since from a documentation perspective it does not add anything. You can just use an imperative "In order to unlist a specific package version, follow these steps:".

Also, what do you think about:

1. Describing with a sentence or two what unlisting does (it hides the package version from search and from nuget.org package details page)
1. Linking to the deprecation document, because that's also interesting for people that can't delete
   - https://docs.microsoft.com/en-us/nuget/nuget-org/deprecate-packages
   - Make sure to use a relative link to this other doc, as shown elsewhere in this repo

Reply:
As for why someone would unlist a package: it allows existing users of the package to keep downloading it but reduces new adoption since the package is not visible in search.